unit_tests/test_microbenchmarking.py

```diff
@@ -167,9 +170,11 @@ def test_generate_html_report_for_avg_aio_with_zero_value(self):
         self.assertTrue(os.path.exists(report_file))
         self.assertGreater(os.path.getsize(report_file), 0)
         self.assertTrue(report_html)
-        self.assertIn('large-partition-forwarding', report_html)
-        # no regression for small-partition-slicing tests
-        self.assertNotIn('small-partition-slicing', report_html)
+        for key in report_results.keys():
```

Comment:
it looks like this is the same code from above (copy-paste)

Reply:
Fixed. Tests are passed